test/utils/update/trust.model.fetch.spec.ts

```diff
@@ -665,27 +666,28 @@ describe("Test fetching and mapping of Trust data", () => {
     expect(trust.HISTORICAL_BO).toEqual(undefined);
   });
 
+  const trustMock: Trust = {
```

Comment:
happy to approve, but was thinking maybe in session.mocks since there is no current trustMock with ch_references there at the current moment

Reply:
just resolve this if you think the current is fine!